src/services/faculty-job.ts

```diff
@@ -1,10 +1,10 @@
 import axios from 'axios';
 
-const JOBS_API_URL = 'http://localhost:9000/faculty-jobs';
-const token = localStorage.getItem('token');
-
+const JOBS_API_URL = 'http://localhost:9000/jobs/faculty';
```

Comment:
Please don't change the global variable, it might be used elsewhere. You can create a new one, for example, JOBS_FACULTY_API_URL